_posts/2021-05-10-helm-charts-for-wildfly.adoc

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: Helm Charts for WildFly
+date: 2021-05-10
+tags: wildfly openshift helm
+synopsis: Helm Charts for WildFly simplifies deployment of WildFly applications on OpenShift.
+author: jmesnil
+---
+
+https://helm.sh[Helm] is a tool that help define, install and upgrades complex applications on Kubernetes clusters.
```

Comment:
s/is a tool that help/is a tool that helps
s/install and upgrades/install and upgrade